stable/datadog/templates/container-trace-agent.yaml

```diff
@@ -14,6 +14,8 @@
     {{- include "containers-common-env" . | nindent 4 }}
     - name: DD_LOG_LEVEL
       value: {{ .Values.agents.containers.traceAgent.logLevel | default .Values.datadog.logLevel | quote }}
+    - name: DD_APM_ENABLED
+      value: "true"
```

Comment:
which agent version are you using? 
I think the default value is now `true` by default: https://github.com/DataDog/datadog-agent/blob/1c76b8381a195a0b0f629011a6225e936fe1d37a/pkg/config/config.go#L390

Reply:
I believe it to address this issue:

```bash
2020-03-07 00:16:32 UTC | TRACE | INFO | (pkg/util/log/log.go:482 in func1) | Loaded configuration: /etc/datadog-agent/datadog.yaml
2020-03-07 00:16:32 UTC | TRACE | INFO | (pkg/trace/agent/run.go:71 in Run) | trace-agent not enabled. Set the environment variable
DD_APM_ENABLED=true or add "apm_config.enabled: true" entry
to your datadog.yaml. Exiting...
```

Reply:
I can confirm that in 7.18.0-rc.6 it's on by default, however.